test/test_manifest.json

```diff
@@ -967,6 +967,21 @@
        "enableXfa": true,
        "type": "eq"
     },
+    {  "id": "xfa_bug1729877",
+       "file": "pdfs/xfa_bug1729877.pdf",
+       "md5": "1bf5429ea13a090f52edcd49cb0c4a47",
+       "link": true,
```

Comment:
Are we allowed to include this document in the repo?
 - If no, please update the patch to actually use a *linked* test-case instead.
 - If yes, this line should simply be removed.